lib/sunrise/sunrise.php

```diff
@@ -9,7 +9,9 @@
 	return;
 }
 
-require_once WP_CONTENT_DIR . '/mu-plugins/lib/utils/class-context.php';
+$mu_plugin_dir = defined( 'WPMU_PLUGIN_DIR' ) ? constant( 'WPMU_PLUGIN_DIR' ) : WP_CONTENT_DIR . '/mu-plugins';
```

Comment:
```suggestion
$mu_plugin_dir = defined( 'WPMU_PLUGIN_DIR' ) ? constant( 'WPMU_PLUGIN_DIR' ) : constant( 'WP_CONTENT_DIR' ) . '/mu-plugins';
```
For consistency if we are using `constant()`